commands.go

```diff
@@ -327,12 +331,16 @@ func cdCMD(rest string, u *user) {
 		}
 	}
 	if strings.HasPrefix(rest, "#") {
-		u.room.broadcast(u.name, "cd "+rest)
-		if v, ok := rooms[rest]; ok {
+		newRoom := rest
+		if len(newRoom) > maxLengthRoomName {
+			newRoom = newRoom[0:maxLengthRoomName]
```

Comment:
Do you think we should send a message that room names are only allowed to be less than a certain length?

Reply:
This is a really good idea. I will implement it.